packages/mobile/src/utils/permissions.ios.ts

```diff
@@ -0,0 +1,40 @@
+import Contacts from 'react-native-contacts'
+
+export async function requestPhoneStatePermission() {
+  throw new Error('Unsupported method')
+  return false
+}
+
+export async function requestContactsPermission(): Promise<boolean> {
+  return new Promise((resolve, reject) => {
+    Contacts.requestPermission((err, permission) => {
+      if (err) {
+        reject(err)
+      } else {
+        resolve(permission === 'authorized')
+      }
+    })
+  })
+}
+
+export async function requestCameraPermission() {
+  throw new Error('Unsupported method')
+  return false
+}
+
+export async function checkContactsPermission(): Promise<boolean> {
+  return new Promise((resolve, reject) => {
+    Contacts.checkPermission((err, permission) => {
+      if (err) {
+        reject(err)
+      } else {
+        resolve(permission === 'authorized')
+      }
+    })
+  })
+}
+
+export async function checkCameraPermission() {
+  throw new Error('Unsupported method')
```

Comment:
Are these no possible on iOS or just not yet implemented?

Reply:
Good catch, I need to rename them to `not implemented`